### PR TITLE
bump version 3.8.0

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -19,6 +19,11 @@
   <change-notes>
     <![CDATA[
     <html>
+      <h3>[3.8.0]</h3>
+      <ul>
+        <li>Supported to run Spark jobs in Azure Data Lake cluster.</li>
```
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Is this line OK?  @djlan @wezhang

Comment (djlan) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: Please add "(in private preview)".

Comment (Eskibear) [reply PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml]: @djlan updated, please verify.

**Comment by Eskibear** [comment]
rerun tests.

**Comment by Eskibear** [comment]
rerun tests.

**Pull request merged**